src/datatable/datatable.jl

```diff
@@ -214,9 +212,8 @@ function Base.getindex(dt::DataTable, col_ind::ColumnIndex)
 end
 
 # dt[MultiColumnIndex] => DataTable
-function Base.getindex{T <: ColumnIndex}(dt::DataTable,
-                                         col_inds::Union{AbstractVector{T},
-                                                         AbstractVector{Nullable{T}}})
+function Base.getindex{T <: ?ColumnIndex}(dt::DataTable,
+                                         col_inds::AbstractVector{T})
```

Comment:
The addition of the `?` is throwing off indentation here and for a few functions below